### PR TITLE
Bumped recipe version to 1.10.1

### PR DESCRIPTION
**Pull request by jlstevens**
Bump for the 1.10.1 bug fix release.

### CODE REVIEW

**Comment by jakirkham** [comment]
Would reset build number to `0` as well. Were there any dependency changes?

**Comment by jlstevens** [comment]
Odd that the pinning fails with what looks like a syntax error regarding the use of ``=``. I just copied the lines for pinning dependencies from the [meta.yaml](https://github.com/ioam/holoviews/blob/master/conda.recipe/meta.yaml) in the repo which I know works fine...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,27 +7,28 @@ package:
...
     - pip
+    - param >=1.6.0,<2.0
   run:
     - python
-    - param
+    - param >=1.6.0,<2.0
     - numpy
     - matplotlib
-    - bokeh
+    - bokeh>=0.12.14,<=0.12.16
```
Comment (jakirkham) [inline recipe/meta.yaml]: Needs a space after the package name and before `>=`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,27 +7,28 @@ package:
...
     - jupyter
     - notebook
-    - ipython
+    - ipython>=5.4.0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Needs a space after the package name and before `>=`.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Added some comments above to clarify what issue the CI was running into. However as it appears cb3 relaxes the constraints, went ahead and re-rendered to get cb3, which also solves the problem.

**Comment by jakirkham** [comment]
Even though using cb3 worked. It sounds like that isn't intentional behavior. So you may still want to add spaces after the package names anyways.

**Comment by jlstevens** [comment]
> Even though using cb3 worked. It sounds like that isn't intentional behavior. So you may still want to add spaces after the package names anyways.

Thanks for the tip! I wrongly assumed that because it worked for me locally, it was supported as part of the spec.

**Comment by jlstevens** [comment]
Anyway, I've fixed the pinning syntax in the meta.yaml both here and on holoviews master.

**Comment by jakirkham** [comment]
LGTM. Thanks @jlstevens.

**Comment by jakirkham** [comment]
> Even though using cb3 worked. It sounds like that isn't intentional behavior. So you may still want to add spaces after the package names anyways.

FWIW raised issue ( https://github.com/conda/conda-build/issues/2828 ) on this point.

**Pull request merged**